cmd/action.go

```diff
@@ -0,0 +1,267 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/salt/log"
+	"github.com/odpf/salt/printer"
+	"github.com/odpf/shield/config"
+	shieldv1beta1 "github.com/odpf/shield/proto/v1beta1"
+	cli "github.com/spf13/cobra"
+)
+
+func ActionCommand(logger log.Logger, appConfig *config.Shield) *cli.Command {
+
+	cmd := &cli.Command{
+		Use:     "action",
+		Aliases: []string{"action"},
+		Short:   "Manage action",
+		Long: heredoc.Doc(`
+			Work with actions.
+		`),
+		Example: heredoc.Doc(`
+			$ shield action create
+			$ shield action update
+			$ shield action get
+			$ shield action list
+		`),
+		Annotations: map[string]string{
+			"action:core": "true",
+		},
+	}
+
+	cmd.AddCommand(createActionCommand(logger, appConfig))
+	cmd.AddCommand(updateActionCommand(logger, appConfig))
+	cmd.AddCommand(getActionCommand(logger, appConfig))
+	cmd.AddCommand(listActionCommand(logger, appConfig))
+
+	return cmd
+}
+
+func createActionCommand(logger log.Logger, appConfig *config.Shield) *cli.Command {
+	var filePath, header string
+
+	cmd := &cli.Command{
+		Use:   "create",
+		Short: "create all actions",
```

Comment:
@ishanarya0 Description would be "Create an action". First letter cap. We need to follow similar convention across all descriptions. Check Guardian for ref.